.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
add .idea to your .gitignore (IntelliJ files should not end up in Github)
make sure your .gitignore is on the right level within your file structure

Reply:
Thanks!